src/main/features/claim/views/claimant-mobile.njk

```diff
@@ -1,13 +1,14 @@
 {% extends "layout.njk" %}
 {% from "form.njk" import csrfProtection, input, saveAndContinueButton, errorSummary %}
 
-{% set heading = 'Enter a contact number (optional)' %}
+{% set heading = 'Enter a phone number (optional)' %}
```

Comment:
This will require the change in route test as well

Reply:
done